engine/src/main/java/org/terasology/rendering/logic/MeshRenderer.java

```diff
@@ -244,6 +242,21 @@ private void renderEntitiesByMaterial(SetMultimap<Material, EntityRef> meshByMat
         }
     }
 
+    /**
+     * Checks whether the entity at the given position is relevant.
+     * <p>
+     * The entity is at the given position is relevant if
```

Comment:
The first "is" should be removed I think. Should read "The entity at the given position...", right?